src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -217,6 +218,13 @@ def load_arguments(self, _):
                    arg_group='Autoscale Configuration',
                    help='The scheduled workernode count.')
 
+        # relay outbound and private link
+        c.argument('resource_provider_connection', options_list=['--resource-provider-connection', '--rp-connection'],
+                   arg_group='Relay Outbound And PrivateLink',
+                   arg_type=get_enum_type(ResourceProviderConnection), help='The resource provider connection type')
+        c.argument('private_link', arg_group='Relay Outbound And PrivateLink',
```

Comment:
I don't quite understand this design. From the perspective of the argument name, it looks like Relay bound is a subproperty of PrivateLink. But from the perspective of the argument group name, they are equipotent resources.

Reply:
Hi @haroldrandom I have updated the PR could you please review it again? Thanks